.env

```diff
@@ -1,3 +1,3 @@
-REACT_APP_API_URL = http://localhost:3000
-REACT_APP_GOOGLE_REDIRECT = `${REACT_APP_API_URL}/auth/redirect`
+REACT_APP_API_URL = 'http://localhost:3000'
+REACT_APP_GOOGLE_REDIRECT = 'http://localhost:3000/auth/redirect'
```

Comment:
this file should be in gitignore, please do not push

Reply:
deleted